website/docs/cloud-docs/policy-enforcement/define-policies/prewritten-sentinel.mdx

```diff
@@ -0,0 +1,129 @@
+This method enables you to connect the policies to workspaces in the Terraform configuration file. As a result, you can skip the instructions described in [Connect policies to your workspaces](#connect-policies-to-your-workspaces). 
+	 
+1. Go to the [module in the Terraform registry](https://registry.terraform.io/modules/hashicorp-modules/aws-cis/sentinel/latest) and copy the code generated in the **Provision Instructions** tile.  
+1. Add the `module` block to your Terraform configuration and define the following arguments: 
+   - `source`: Specify the path to the module you downloaded.
+   - `tfe_organization`: Specify the name of your organization on Terraform Enterprise or HCP Terraform.
+   - `policy_set_workspace_names`: Specify a list of workspace names that you want to apply the policies to. 
+
+   The following example configuration applies invokes the module for `target_workspace_1`:
+
+   ```hcl
+   module "cis_v1-2-0_policies" {
+      source = "../prewritten-policy"
+      name                             = "cis-1-2-0"
+      tfe_organization                 = "<your-tfe-org>"
+      policy_set_workspace_names       = ["target_workspace_1"]
+   }
+   ```
+
+1. Run `terraform plan` to view the plan.
+1. Run `terraform apply` to apply the changes.
+
+After successfully running the `terraform apply` command, Terraform evaluates Sentinel policies for each run of the workspaces you specified.
+
+## Connect policies to your workspace
+
+Skip this step if you [added the Terraform module](#add-the-terraform-module-to-your-configuration) to your configuration. When you use the module, you programmatically connect policies to workspaces in the HCL. 
+
+1. Log into your organization and click **Settings** in the sidebar. 
+1. Click **Policy Sets** and click **Connect a new policy set**.
+1. Click the **Version control provider (VCS)** tile.
+1. Enable the **Sentinel** option as the policy framework.
+1. Specify a name and description for the set.
+1. Configure any additional options for the policy set and click **Next**.
+1. Choose the GitHub connection type, then choose the repository you created in [Set up a repository for the policies](#set-up-a-repository-for-the-policies). 
+1. If you stored the `sentinel.hcl` policy set file in folder, specify the path to the file in the **Policies path** field. The default is the root directory.
+1. If you want to apply updated policies sets to the workspace from a specific branch, specify the name in the **VCS branch** field.  The default is the default branch configured for the repository.
+1. Click **Next** and specify any additional parameters you want to pass to the Sentinel runtime and click **Connect policy set** to finish applying the policies to the workspace. 
+
+Run a plan in the workspace to trigger the connected policies. Refer to [Start a Terraform run](/terraform/cloud-docs/run/remote-operations#starting-runs) for additional information.
```

Comment:
It might be nice to explain why and when you would do this, eg: "Some policies include configuration parameters to customize their behavior. For example, the CIS repository used in the previous example includes parameters to..."